### PR TITLE
stripe docs

### PR DESCRIPTION
**Pull request by jpjoyal**
Will re-update screenshots once the live-test mode switch is in.

I'm sure I'm missing important stuff here, need help getting more content.

@alavers @mspensieri @gozman

### CODE REVIEW

File: source/includes/integrations/_stripe.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Use Test mode during development, so you can use fake [credit card numbers](https://stripe.com/docs/testing#cards). Select Live to get real money 🤑.
+
+Then Click "Authenticate your Stripe account". This will send you to a Stripe page where you can select which Stripe account to link Smooch with. 
+
+<aside class="info">
+If you are using test mode, notice the "Skip this account form" a the top of the Stripe page. No need for a bank account for test mode.
+</aside>
+
+### Asking for money
```
Comment (mspensieri) [inline source/includes/integrations/_stripe.md]: Should we also mention that credit card information is never handled by our servers?

File: source/includes/integrations/_stripe.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+This works from any channel you configured to use with Smooch.
+
+The amount you specify must be using the dollars dot cents syntax (ie 25.33 for 25 dollars and 33 cents).
+
+Once the user completes a payment you'll get a confirmation along with a link to the payment page on Stripe.
+
+
+
```
Comment (mspensieri) [inline source/includes/integrations/_stripe.md]: Can we add a section about the sms / old sdk experience?

File: source/includes/integrations/_stripe.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+While inside a conversation with a user you can send a buy button with the following syntax
+
+```
+$[Button label here](25.00)
+```
+
+This works from any channel you configured to use with Smooch.
+
+The amount you specify must be using the dollars dot cents syntax (ie 25.33 for 25 dollars and 33 cents).
```
Comment (mspensieri) [inline source/includes/integrations/_stripe.md]: The user can also specify round numbers `$[Text](25)`

File: source/includes/integrations/_stripe.md (hunk 1 above)
Comment (dannytranlx) [inline source/includes/integrations/_stripe.md]: Can we find better copy for `Asking for money`? Something something `Accepting payments`, `Requesting payments`?

File: source/includes/integrations/_stripe.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	<img alt="payment completed ui" src="../images/payment_completed_ui.png" />
+</div>
+
+### Configuring Stripe
+
+From the Stripe integration page, select which Stripe mode you want Smooch to be using. 
+
+![Stripe integration page ui](Stripe_integration_page_ui.png)
+
+Use Test mode during development, so you can use fake [credit card numbers](https://stripe.com/docs/testing#cards). Select Live to get real money 🤑.
```
Comment (dannytranlx) [inline source/includes/integrations/_stripe.md]: > you can use _test_ credit card numbers?

File: source/includes/integrations/_stripe.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Then Click "Authenticate your Stripe account". This will send you to a Stripe page where you can select which Stripe account to link Smooch with. 
+
+<aside class="info">
+If you are using test mode, notice the "Skip this account form" a the top of the Stripe page. No need for a bank account for test mode.
```
Comment (dannytranlx) [inline source/includes/integrations/_stripe.md]: > at* the top

File: source/includes/integrations/_stripe.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+## Stripe
+
+Stripe is a developer-friendly platform that instantly enables businesses to accept and manage payments online and in mobile apps. Our Stripe integration allows you to ask money to your users and accept payments directly within the conversation. Credit card informations is never seen by our servers, everything related to transactions is all handled by Stripe.
```
Comment (dannytranlx) [inline source/includes/integrations/_stripe.md]: > Credit card informations is

Cannot grammar, one should be plural, or the other should be singular.

Comment (alavers) [inline source/includes/integrations/_stripe.md]: Both singular,

> Credit card information is

Also "everything related to transactions is all handled by Stripe" would be more truthfully told as:

> Credit card information is never seen by our servers, Stripe processes all credit card transactions.

File: source/includes/integrations/_stripe.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+## Stripe
+
+Stripe is a developer-friendly platform that instantly enables businesses to accept and manage payments online and in mobile apps. Our Stripe integration allows you to ask money to your users and accept payments directly within the conversation. Credit cards informations are never seen by our servers, everything related to transactions is all handled by Stripe.
```
Comment (alavers) [inline source/includes/integrations/_stripe.md]: > Stripe integration allows you to request payment from your users directly within the conversation

File: source/includes/integrations/_stripe.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+	<img alt="card capture ui" src="../images/card_capture_ui.png" />
+	<img alt="payment completed ui" src="../images/payment_completed_ui.png" />
+</div>
+
+### Configuring Stripe
+
+From the Stripe integration page, select which Stripe mode you want Smooch to be using. 
+
+![Stripe integration page ui](Stripe_integration_page_ui.png)
+
```
Comment (alavers) [inline source/includes/integrations/_stripe.md]: suggest:

> Use Test mode during development. While in this mode you can use Stripe's [test credit card numbers](https://stripe.com/docs/testing#cards) to make transactions without using real money.

File: source/includes/integrations/_stripe.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+Use Test mode during development, so you can use test [credit card numbers](https://stripe.com/docs/testing#cards). Select Live to get real money 🤑.
+
+Then Click "Authenticate your Stripe account". This will send you to a Stripe page where you can select which Stripe account to link Smooch with. 
+
+<aside class="info">
+If you are using test mode, notice the "Skip this account form" at the top of the Stripe page. No need for a bank account for test mode.
```
Comment (alavers) [inline source/includes/integrations/_stripe.md]: > No need for a bank account in test mode.

**Comment by alavers** [comment]
Might be worthwhile to mention somewhere that all amounts are in USD.

File: source/includes/integrations/_stripe.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+This works from any channel you configured to use with Smooch.
+
+The amount you specify can be just a dollars amount or dollars and cents as in the example.
+
+Once the user completes a payment you'll get a confirmation along with a link to the payment page on Stripe.
+
+<aside class="info">
+If your users are messaging you by SMS via our [Twillo](#twilio) integration, they'll be getting a web link instead of a buy button. It will take them to a payment web page where we use [Checkout.js](https://stripe.com/docs/checkout) from Stripe to secure the payment. This mechanism is also used for versions prior to 3.3 of the SDK.
```
Comment (alavers) [inline source/includes/integrations/_stripe.md]: nit:

> This mechanism is also used for SDK versions older than 3.3.

**Comment by alavers** [comment]
Rest looks :+1:

File: source/includes/integrations/_stripe.md (hunk 10 above)
Comment (mspensieri) [inline source/includes/integrations/_stripe.md]: should specify iOS. This mechanism is _still_ used by android and web.

**Comment by mspensieri** [comment]
Should say somewhere that we take a percentage of purchases

**Comment by mspensieri** [comment]
:+1:

**Pull request merged**